cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/configuration.rst

```diff
@@ -0,0 +1,107 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: HTTP RESTful Interface to the Cask Data Application Platform
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. highlight:: console
+
+.. _http-restful-api-configuration:
+.. _http-restful-api-v3-configuration:
+
+======================================
+Configuration Service HTTP RESTful API
```

Comment:
Is this a service? It may give a wrong notion - should we just call it "Configuration HTTP RESTful API"